src/app/modules/shared_models/user_model.nim

```diff
@@ -140,14 +148,19 @@ QtObject:
     let index = self.createIndex(ind, 0, nil)
     self.dataChanged(index, index, @[ModelRole.Icon.int, ModelRole.IsIdenticon.int])
 
-  proc updateItem*(self: Model, id: string, name: string, icon: string, isIdenticon: bool) = 
+  proc updateItem*(
+    self: Model, id: string, name: string, icon: string, isIdenticon: bool, 
+    isAdmin: bool = false, joined: bool = false
+  ) = 
     let ind = self.findIndexForMessageId(id)
     if(ind == -1):
       return
 
     self.items[ind].name = name
     self.items[ind].icon = icon
     self.items[ind].isIdenticon = isIdenticon
+    self.items[ind].isAdmin = isAdmin
+    self.items[ind].joined = joined
     
     let index = self.createIndex(ind, 0, nil)
     self.dataChanged(index, index, @[ModelRole.Name.int, ModelRole.Icon.int, ModelRole.IsIdenticon.int])
```

Comment:
@alaibe one minor thing remained. Please either add `ModelRole. IsAdmin.int, ModelRole. Joined.int` roles to the list of changed roles of `dataChanged` signal or add new proc here to `user_model.nim` similar to one we already have but for updating other params like this `proc updateItem*(self: Model, id: string, isAdmin: bool = false, joined: bool = false) =` (I think that proc name overloading this way is allowed in Nim) and then in the `chat_section/chat_content/users/module.nim`, method `onChatMemberUpdated` will update only fields which actually were changed:
```
method onChatMemberUpdated*(self: Module, publicKey: string, admin: bool, joined: bool) =
  self.view.model().updateItem(publicKey, admin, joined)
```